### PR TITLE
[R4R] - {0.4.2}: Bugfix/sigma mnt 11

### PR DESCRIPTION
**Pull request by Sha3nS**
# Goals of PR

Core changes:
fix sigma audit MNT-11

Notes:
no

Related Issues:
TBD

### CODE REVIEW

**Review by byteflyfunny — DISMISSED** [review]

File: mt-challenger/challenger/challenger_test.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package challenger
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"testing"
```
Comment (HaoyangLiu) [inline mt-challenger/challenger/challenger_test.go:6]: go format

File: batch-submitter/drivers/sequencer/encoding_test.go [hunk 2]
```diff
@@ -252,3 +255,27 @@ func TestIsMarkerContext(t *testing.T) {
 	}
 	require.True(t, batchContext.IsMarkerContext())
 }
+
+func TestReadUint64(t *testing.T) {
+	readUint64 := func(r io.Reader, val *uint64, n uint) error {
```
Comment (HaoyangLiu) [inline batch-submitter/drivers/sequencer/encoding_test.go:260]: Why do you define `readUint64` again here? Can we just call the `readUint64` in `batch-submitter/drivers/sequencer/encoding.go`?

File: mt-challenger/challenger/challenger_test.go (hunk 1 above)
Comment (Sha3nS) [reply mt-challenger/challenger/challenger_test.go:6]: done

**Review by HaoyangLiu — APPROVED** [review]

File: batch-submitter/drivers/sequencer/encoding_test.go (hunk 2 above)
Comment (Sha3nS) [reply batch-submitter/drivers/sequencer/encoding_test.go:260]: test file is in a different package, can't reference an unexported method

**Review by guoshijiang — APPROVED** [review]

**Pull request merged**